common/lib/capa/capa/templates/codeinput.html

```diff
@@ -4,9 +4,7 @@
 from openedx.core.djangolib.markup import HTML
 %>
 <div id="textbox_${id}" class="capa_inputtype textbox cminput">
-  % if response_data['label']:
-      <label class="problem-group-label" for="cm-textarea-${id}">${response_data['label']}</label>
-  % endif
+  <label class="sr problem-group-label" for="cm-textarea-${id}">Code Editor ${response_data['label']}</label>
```

Comment:
@muzaffaryousaf @muhammad-ammar @cahrens @alisan617 

In out work on Capa the response_data['label'] was added here as a conditional.  I cannot find anywhere for codeinput that this is set in the inputtypes.py file.  Do any of you know its use/purpose?  If not I am going to remove it entirely from the html file.